lib/node_modules/@stdlib/math/base/special/acoth/benchmark/cpp/boost/Makefile

```diff
@@ -0,0 +1,110 @@
+#/
```

Comment:
To my knowledge, Boost does not have an `acoth` function. You can remove this file.

Reply:
This file still needs to be removed.